hardhat.config.ts

```diff
@@ -31,8 +34,35 @@ const DEFAULT_COMPILER_SETTINGS: SolcUserConfig = {
     },
   },
 }
-
-const config: HardhatUserConfig = {
+interface EtherscanConfig {
+  apiKey: {
+    sepolia: string
+    arbitrumSepolia: string
+    arbitrumOne: string
+  },
+  customChains: any
+}
+const EtherscanConfig: EtherscanConfig = {
+  apiKey: {
+    sepolia: ETHERSCAN_API_KEY || "",
+    arbitrumSepolia: ETHERSCAN_API_KEY || "",
+    arbitrumOne: ETHERSCAN_API_KEY || ""
```

Comment:
Why allow empty string?

Reply:
This is because the type defined for these keys are as String. So, if we do not provide the empty strings the TS throws errors that these values can be undefined as well. So, to remove that error these empty strings are added. 

There is another way as well to avoid that error i.e. to define the type as String | undefined, but this will then conflict with the types of hardhat where it requires the strings only for these. That's we can't use String | undefined here, and have added empty strings for that.

Reply:
Gotcha. Thanks for the clarification